src/components/Button/BookmarkButton.js

```diff
@@ -0,0 +1,44 @@
+import useStore from '../useStore/useStore';
+import { WrapperBookmark } from '../Form/Form.styled';
+import MySVG from '../SVG/SVG';
+
+export default function Bookmark() {
+	const updateBookmark = useStore(state => state.updateBookmark);
+	const bookmarkStatus = useStore(state => state.bookmarkStatus);
+	console.log(bookmarkStatus);
+	return (
+		<WrapperBookmark>
+			<p>Liked</p>
+			{bookmarkStatus === 'liked' ? (
+				<div onClick={() => updateBookmark('')}>
+					{' '}
+					<MySVG variant="heartFilled" color="#694370" />
+				</div>
+			) : (
+				<div onClick={() => updateBookmark('liked')}>
+					<MySVG variant="heartEmpty" />
+				</div>
+			)}
+			<p>Disliked</p>
+			{bookmarkStatus === 'disliked' ? (
+				<div onClick={() => updateBookmark('')}>
+					<MySVG variant="brokenHeartFilled" color="#694370" />{' '}
+				</div>
+			) : (
+				<div onClick={() => updateBookmark('disliked')}>
+					<MySVG variant="brokenHeartEmpty" />
+				</div>
+			)}
+			<p>Tbr</p>
+			{bookmarkStatus === 'tbr' ? (
+				<div onClick={() => updateBookmark('')}>
+					<MySVG variant="bookFilled" color="#694370" />{' '}
+				</div>
+			) : (
+				<div onClick={() => updateBookmark('tbr')}>
+					<MySVG variant="bookEmpty" />
+				</div>
+			)}
```

Comment:
Adventurous long ternary-statement, which is surprisingly easy to read.. but a way better choice would have probably been a [switch case](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch). I will let you decide on that, only get rid of the `{' '}` in line 14.

Something you should definitely change is having divs with an onClick, I think you should use buttons for this, due to them being accessible by tabbing on your keyboard.